Enterprise-CMCS/macpro-mdct-qmr
# MDCT-2337: Non-Required Measurement Fix (SPIKE Output)

### Description

In a few instances, state users reported that the auto-completed (i.e. read-only) measures were being set to `In Progress` even though these should always be marked `Complete`. After taking a look with @BearHanded we saw that on a session time-out, the auto-completed measures are getting "saved" and thus returning to the `In Progress` state.

### Related ticket(s)


MDCT-2337

---
### How to test


- Navigate to any FY Adult Measures
- Verify that `NCIDDS-AD` is set to `Complete` by default
- Navigate away and back
- It should still be set `Complete`

### Important updates




---
### Author checklist


---


_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_
